### PR TITLE
Added additional MAC match for Cisco switches.

### PR DESCRIPTION
**Pull request by josephgorse**
Changed compute nodes pool IP range to avoid overlapping range error

### CODE REVIEW

**Comment by rackhdmirror** [comment]
@RackHD/corecommitters please review.

File: dhcpd.conf [hunk 1]
```diff
@@ -70,7 +71,7 @@ subnet 10.1.1.0 netmask 255.255.255.0 {
  }
  # compute nodes
  pool {
-        range 10.1.1.2 10.1.1.230;
+        range 10.1.1.2 10.1.1.220;
```
Comment (benbp) [inline dhcpd.conf]: Seems like an unrelated change (albeit inconsequential)?

**Comment by benbp** [comment]
test this please

**Comment by rackhdmirror** [comment]
Build finished.
